src/index.ts

```diff
@@ -64,7 +64,7 @@ export class Datastore {
         await this._initialize()
 
         const storageId = await this._storage.addFile(file)
-        const fileId = await this._contract.addFile(storageId, name, file.byteLength, true)
+        const fileId = await this._contract.addFile(storageId, name, file.byteLength, false)
```

Comment:
I think I would leave isPublic to true by default since we don't have support for private files right now